go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/babylonlabs-io/babylon-staking-indexer
 go 1.23.2
 
 require (
-	github.com/babylonlabs-io/babylon v0.11.0
+	github.com/babylonlabs-io/babylon v0.12.0
```

Comment:
can change to https://github.com/babylonlabs-io/babylon/commit/86f6bc5d99867f6bf0706fe4ab4c3eee8ae30a06 which includes format change